protected-api-access-protected-api/README.md

```diff
@@ -0,0 +1,142 @@
+- Node.Js 16+
+
+## Setup
+
+### 1. Register the app
+
+Complete the steps in [Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the sample app.
+
+Use these settings in your app registration.
+
+| App registration <br/> setting    | Value for this sample app                                                    | Notes                                                                                              |
+|:---------------------------------:|:-----------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
+| **Name**                          | `Node Web API`                                                               | Suggested value for this sample. <br/> You can change the app name at any time.                    |
+| **Supported account types**       | **Accounts in this organizational directory only (Single tenant)**           | Suggested value for this sample.                                                                   |
+| **Platform type**                 | _None_                                                                      | No redirect URI required; don't select a platform.                                                                    |
+| **Client secret**                 | _**Value** of the client secret (not its ID)_                                | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it).        |
+| **API Permissions**               | `user_impersonation`                                                         | Create a new permission called user_impersonation.  Required value for this sample.      |
+
+
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+
+
+### 2. Update code sample with app registration values
+
+```javascript
+  auth: {
+    // 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+    clientId: '',
+    
+    // Client secret 'Value' (not the ID) from 'Client secrets' in app registration in Azure portal
+    clientSecret: '',
+    
+    // Full directory URL, in the form of https://login.microsoftonline.com/<tenant>
+    authority: ''
+  }
+```
+
+```javascript
+  secret: jwks.expressJwtSecret({
+    // Full URL, in the form of: https://login.microsoftonline.com/<tenant>/discovery/v2.0/keys
+    jwksUri: ''
+  }),
+```
+
+
+### 3. Install package(s)
+
+To install Node.Js and MSAL libraries:
+
+```bash
+npm install
+```
+
+## Run the application
+
+```bash
+node app.js
+```
+
+## Browse to the application
+
+Using postman or a similar app, navigate to **http://localhost:8080/me**
```

Comment:
Using Postman, curl, or a similar application, issue an HTTP GET request to *http://localhost:8080/me* with an `Authorization` header of `Bearer {VALID-ACCESS-TOKEN}`.

Reply:
Updated